example_code/applications/vending_machine/README.md

```diff
@@ -0,0 +1,107 @@
+# Stylus Cupcake Example
+
+An example project for writing Arbitrum Stylus programs in Rust using the [stylus-sdk](https://github.com/OffchainLabs/stylus-sdk-rs). It includes a Rust implementation of a vending machine Ethereum smart contract.
+
+```js
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+// Rule 2: The vending machine's rules can't be changed by anyone.
+contract VendingMachine {
```

Comment:
Hmm, this entire example will be included in the docs as an alternative page for our vending machine using Stylus. I've outlined the Solidity contract here for two reasons: (1) to maintain consistency with our current documentation, and (2) to ensure users understand which contract we're implementing with Stylus.

How about we change the description?
An example project for writing Arbitrum Stylus programs in Rust using the [stylus-sdk](https://github.com/OffchainLabs/stylus-sdk-rs). **It includes a Rust implementation of the following vending machine Ethereum smart contract**.

Reply:
They won't be looking at the Solidity contract though, right? Many users will have never even seen the Solidity cupcake example when they come across this README. This will just live in the Stylus docs. To me, the README is about instructing users for this example, specifically, and it's confusing to see Solidity off the bat when expecting to be coding in Rust.

Reply:
Maybe just define the interface?

Reply:
yup that sounds good